File: Cuaderno_1/ej13.py
"""
13.Se desea conocer el importe en Libras Esterlinas (GBP) al cambio de una
cantidad en Euros (EUR). Escribe un programa que, a partir de una cierta
cantidad en euros y del tipo de cambio del día, muestre el equivalente en
libras teniendo en cuenta que la casa de cambio retiene una comisión del 2%
sobre el total de la operación.

"""

euros = float(input("Introduce la cantidad de EUR: "))

EURaGBP = 0.87 # A 23 de septiembre de 2023

def cambio(eur):
    return euros * EURaGBP * (1-0.02)

print(f"Para {euros}EUR vas a recibir {cambio(euros)}")

File: Cuaderno_1_JAVA/ej13.java
/*
 * euros = float(input("Introduce la cantidad de EUR: "))

EURaGBP = 0.87 # A 23 de septiembre de 2023

def cambio(eur):
    return euros * EURaGBP * (1-0.02)

print(f"Para {euros}EUR vas a recibir {cambio(euros)}")
 */
import java.util.Scanner;
public class ej13 {
    public static void main(String []args) {
        Scanner read = new Scanner(System.in);
        System.out.println("Introduce la cantidad de EUR");
        float euros = Float.parseFloat(read.nextLine());
        read.close();
        float cambio = cambio(euros);
        System.out.println("El cambio de la cantidad introducida es de "+cambio+" GBP");
    }

    public static float cambio(float eur) {
        float EURaGBP = 0.87f;
        return eur * EURaGBP * (1-0.02f);
    }
}
